odoo/addons/openupgrade_records/lib/apriori.py

```diff
@@ -75,4 +75,6 @@
     'ir.actions.report.xml': 'ir.actions.report',
     'stock.pack.operation': 'stock.move.line',
     'stock.picking.wave': 'stock.picking.batch',
+    # OCA/partner-contact
```

Comment:
I think you should put a longer comment indicating the reason for this.

Reply:
ok 😑

Reply:
done